### PR TITLE
fixed decorators

### PR DESCRIPTION
**Pull request by Velrok**

### CODE REVIEW

File: wobble/__init__.py [hunk 1]
```diff
@@ -11,19 +11,28 @@ class WobbleService(object):
...
-        return wrapper
+            return fn(self, *args)
+        return requires_login_decorator
 
     def log_calls(fn):
-        def wrapper(self, *args):
-            result = fn(self, args)
+        def log_calls_decorator(*params):
+            self = params[0]
+            args = params[1:]
```
Comment (zeisss) [inline wobble/__init__.py:23]: so, what happens if there is no second parameter?

File: wobble/__init__.py [hunk 2]
```diff
@@ -11,19 +11,28 @@ class WobbleService(object):
...
+
+            logging.debug("{obj}.{method}{params}".format(
```
Comment (zeisss) [inline wobble/__init__.py:25]: paranthesis?

Comment (Velrok) [inline wobble/__init__.py:25]: Are included by default, because params is a tuple wich prints ->
``(arg1, arg2)`

File: wobble/__init__.py (hunk 1 above)
Comment (Velrok) [inline wobble/__init__.py:23]: Everything works as it should :) . Python is awesome :) :

```
In [2]: a
Out[2]: ('a',)

In [3]: a[1:]
Out[3]: ()
```

File: wobble/__init__.py (hunk 2 above)
Comment (Uepsilon) [inline wobble/__init__.py:25]: i guess he means why there is a `(` 25-EOL instead of `)` but it's okay - view full changes / file.

**Pull request merged**